components/organisms/MujeresDeOracion/MujeresDeOracion.module.css

```diff
@@ -0,0 +1,150 @@
+.MujeresDeOracion {
+  width: 100%;
+  height: 100%;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+.MujeresDeOracion_welcome {
+  margin-top: 6.25rem;
+  width: 100%;
+  height: 80vh;
+  object-fit: cover;
+  object-position: 50% 50%;
+  background-image: linear-gradient(
+      rgba(255, 97, 97, 0.716),
+      rgba(255, 97, 97, 0.716)
+    ),
+    url("/gruposDesafiantes/mujeresDeOracion/mujeresdeoracion.webp");
+  background-size: cover;
+  background-position: center;
+  background-repeat: no-repeat;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  gap: 0.625rem;
+}
+
+.MujeresDeOracion-title_welcome {
+  color: var(--white);
+  font-size: var(--fontSize-superTitles_mobile);
+  text-align: center;
+}
+
+.MujeresDeOracion-paragraph_welcome {
+  color: var(--white);
+  font-size: var(--fontSize-paragraph_mobile);
+  text-align: center;
+  width: 15.625rem;
+}
+
+.MujeresDeOracion-buttons {
+  width: 100%;
+  height: 30vh;
+  margin-top: 1.25rem;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  gap: 0.625rem;
+}
+
+.MujeresDeOracion-buttons_bgTrasnsparent {
```

Comment:
![image](https://github.com/generaciondesafiante/Regresemos_a_casa_UI/assets/104586980/346bf9b9-1a26-4186-8ea7-0fff515f1cc3)
Estos botones deben tener el border

Reply:
added the button border